org.videolan.VLC.json

```diff
@@ -0,0 +1,867 @@
+        }
+      ]
+    },
+    {
+      "name": "libsndfile",
+      "config-opts": [
+        "--prefix=/app",
+        "--disable-sqlite"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.mega-nerd.com/libsndfile/files/libsndfile-1.0.27.tar.gz",
+          "sha256": "a391952f27f4a92ceb2b4c06493ac107896ed6c76be9a613a4731f076d30fac0"
+        }
+      ]
+    },
+    {
+      "name": "libraw1394",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.kernel.org/pub/linux/libs/ieee1394/libraw1394-2.1.2.tar.xz",
+          "sha256": "03ccc69761d22c7deb1127fc301010dd13e70e44bb7134b8ff0d07590259a55e"
+        }
+      ]
+    },
+    {
+      "name": "libavc1394",
+      "config-opts": [
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/libavc1394/libavc1394/libavc1394-0.5.4.tar.gz",
+          "sha256": "7cb1ff09506ae911ca9860bef4af08c2403f3e131f6c913a2cbd6ddca4215b53"
+        }
+      ]
+    },
+    {
+      "name": "tdb",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.samba.org/ftp/tdb/tdb-1.3.12.tar.gz",
+          "sha256": "60134e32253cac8e2efe5e0185d20123c208bcf6ad15edf2f50d80daadf8c348"
+        }
+      ]
+    },
+    {
+      "name": "pulseaudio",
+      "config-opts": [
+        "--prefix=/app",
+        "--libexecdir=/app/lib",
+        "--with-database=tdb",
+        "--disable-tcpwrap",
+        "--disable-bluez4",
+        "--disable-samplerate",
+        "--disable-rpath",
+        "--disable-default-build-tests",
+        "--without-caps"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://freedesktop.org/software/pulseaudio/releases/pulseaudio-9.0.tar.xz",
+          "sha256": "c3d3d66b827f18fbe903fe3df647013f09fc1e2191c035be1ee2d82a9e404686"
+        }
+      ]
+    },
+    {
+      "name": "alsa-lib",
+      "config-opts": [
+        "--disable-maintainer-mode",
+        "--enable-shared",
+        "--disable-resmgr",
+        "--enable-rawmidi",
+        "--enable-seq",
+        "--enable-aload"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.osuosl.org/pub/blfs/conglomeration/alsa-lib/alsa-lib-1.1.2.tar.bz2",
+          "sha256": "d38dacd9892b06b8bff04923c380b38fb2e379ee5538935ff37e45b395d861d6"
+        }
+      ]
+    },
+    {
+      "name": "libgpg-error",
+      "config-opts": [
+        "--enable-threads"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.osuosl.org/pub/blfs/conglomeration/libgpg-error/libgpg-error-1.25.tar.bz2",
+          "sha256": "f628f75843433b38b05af248121beb7db5bd54bb2106f384edac39934261320c"
+        }
+      ]
+    },
+    {
+      "name": "libgcrypt",
```

Comment:
After using libcrypt from Sdk, I have the following error:

```
checking for libgcrypt-config... /usr/bin/libgcrypt-config
checking for LIBGCRYPT - version >= 1.4.0... ERROR: /usr/bin/libgcrypt-config should not be used, use an alternative such as pkg-config
./configure: line 14067: test: --should-not-have-used-/usr/bin/libgcrypt-config: integer expression expected
./configure: line 14070: test: --should-not-have-used-/usr/bin/libgcrypt-config: integer expression expected
no
```
I will try to debug this, but just FYI ;)